app/helpers/view_helper.rb

```diff
@@ -21,6 +21,12 @@ def govuk_button_link_to(body, url, html_options = {})
     link_to(body, url, role: 'button', class: html_options[:class], 'data-module': 'govuk-button', draggable: false)
   end
 
+  def select_nationality_options
+    [
+      OpenStruct.new(id: '', name: t('application_form.personal_details.nationality.default_option')),
+    ] + NATIONALITIES.map { |nationality| OpenStruct.new(id: nationality, name: nationality) }
```

Comment:
Do we need `id`s here yet? (it feels disorderly to have ids with spaces, caps etc)